src/cmd/bundle.test.js

```diff
@@ -1,4 +1,6 @@
 const path = require('path');
+const sinon = require('sinon');
+const sandbox = sinon.createSandbox();
```

Comment:
Sinon is a sandbox by itself. You can do `sinon.stub` then in the `afterEach` do `sinon.restore`

Reply:
Point noted. Actually I forgot to remove them after some testing so removed now.